### PR TITLE
Belmer (May 9 Scribe)

### PR DESCRIPTION
**Pull request by cmd-save**
Here is the summary of our meeting on May 9, 2017

### CODE REVIEW

File: past-meetings.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+# May 9, 2017 @ {dna:micro}
+
+## Updates
+Just anything about anyone in their respective organizations.
+
+## Introduction by Garik
+Garik is the new member of the group. He is the CEO of Blvnp Inc, a publishing company. Garik main strength is actually on the area of Finance & Marketing but he is very interested on Software. He has a bit of programming background too using PHP. And most importantly he loves Island Hoping.. ;-)
+
+Blvnp Inc. has a unique culture I must say.
```
Comment (mknecht) [inline past-meetings.md]: hahaha, indeed. A cult-ure.

**Review by mknecht — COMMENTED** [review]
@belmer could you remove the .DSStore file? And maybe add it to the .gitignore ?

**Comment by mknecht** [comment]
btw @belmer added attendees

**Comment by markjeee** [comment]
@belmer there's still a conflict in file, can you rebase your change to latest master?

**Review by markjeee — APPROVED** [review]
Looks good

**Pull request merged**